Libs/vtkSegmentationCore/vtkSegmentation.cxx

```diff
@@ -1573,13 +1573,24 @@ void vtkSegmentation::SeparateSegmentLabelmap(std::string segmentId)
     thresholdErase->SetInValue(0);
     thresholdErase->ReplaceOutOff();
     thresholdErase->Update();
+
```

Comment:
I've just noticed above in line 555: `if (labelmap)...` - why don't we return immediately if `labelmap == nullptr` ?